taipy/core/notification/core_event_consumer.py

```diff
@@ -32,10 +32,18 @@ def process_event(self, event: Event):
             print(f"Received event created at : {event.creation_date}")
             pass
 
-    consumer = MyEventConsumer("consumer_1", event_queue)
+    registration_id, registered_queue = Notifier.unregister(
+        entity_type=EventEntityType.CYCLE,
+        entity_id="CYCLE_cycle_1",
+        operation=EventOperation.CREATION
```

Comment:
This topic will never generate an event. We are registering to "the creation of a cycle whose id is CYCLE_cycle_1". Either the cycle already exists, and the event won't be produced, or it will not, and the randomly generated unique ID won't be the one provided.

I would use a more common use case. Why not registering to the creation of any scenario?

Reply:
AH yes I totally forgot that! I have changed it to:
```
registration_id, registered_queue = Notifier.register(
        entity_type=EventEntityType.SCENARIO,
        operation=EventOperation.CREATION
    )
```